tests/baselines/reference/es6ExportEqualsInterop.errors.txt

```diff
@@ -108,19 +108,19 @@ tests/cases/compiler/main.ts(106,15): error TS2498: Module '"class-module"' uses
     // namespace import
     import * as y1 from "interface";
                         ~~~~~~~~~~~
-!!! error TS2497: Module '"interface"' resolves to a non-module entity and cannot be imported using this construct.
+!!! error TS2497: When writing ECMAScript imports, callable 'export ='-style modules can only be imported by turning on the 'esModuleInterop' flag and using a default import.

```

Comment:
I think this error is now somewhat misleading. What about

```
This module can only be imported with ECMAScript imports by turning on the `esModuleInterop` flag and using a default import.
```

Reply:
@DanielRosenwasser changed :+1:

Reply:
@DanielRosenwasser changed. could you review?

Reply:
@DanielRosenwasser changed. could you review?